cinema/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
The `tests.py` file is empty. Consider adding test cases for the `Movie` model, views, and serializers to ensure the application's reliability and correctness.